src/modules/user/specs/user.energy.compute.service.spec.ts

```diff
@@ -0,0 +1,199 @@
+                LockedAssetService,
+                FarmAbiFactory,
+                FarmFactoryService,
+                WeekTimekeepingAbiServiceProvider,
+                WeeklyRewardsSplittingAbiServiceProvider,
+                WeekTimekeepingComputeService,
+                WeeklyRewardsSplittingComputeService,
+                UserMetaEsdtService,
+                UserEsdtComputeService,
+                StakingProxyService,
+                StakingProxyAbiService,
+                SimpleLockAbiServiceProvider,
+                SimpleLockService,
+                StakingAbiServiceProvider,
+                StakingServiceProvider,
+                PriceDiscoveryServiceProvider,
+                PriceDiscoveryAbiServiceProvider,
+                PriceDiscoveryComputeServiceProvider,
+                EnergyAbiServiceProvider,
+                ProxyService,
+                ProxyPairAbiServiceProvider,
+                ProxyFarmAbiServiceProvider,
+                UserMetaEsdtComputeService,
+                WrapAbiServiceProvider,
+                MXDataApiServiceProvider,
+                MXApiServiceProvider,
+                MXProxyServiceProvider,
+                LockedAssetGetterService,
+                RemoteConfigGetterServiceProvider,
+                AbiLockedAssetServiceProvider,
+            ],
+            imports: [CommonAppModule, CachingModule],
+        }).compile();
+    });
+
+    it('should be defined', () => {
+        const UserEnergyCompute = module.get<UserEnergyComputeService>(UserEnergyComputeService);
+
+        expect(UserEnergyCompute).toBeDefined();
+    });
+
+    it('same values', () => {
+        const UserEnergyCompute = module.get<UserEnergyComputeService>(UserEnergyComputeService);
+
+        expect(UserEnergyCompute).toBeDefined();
+        expect(UserEnergyCompute.isEnergyOutdated({
+            amount: '100',
+            lastUpdateEpoch: 10,
+            totalLockedTokens: '5',
+        }, {
+            week: 10,
+            energy: {
+                amount: '100',
+                lastUpdateEpoch: 10,
+                totalLockedTokens: '5',
+            }
+        })).toBe(false);
+    });
+    it('current user energy values are older', () => {
+        const UserEnergyCompute = module.get<UserEnergyComputeService>(UserEnergyComputeService);
+
+        expect(UserEnergyCompute).toBeDefined();
+        expect(UserEnergyCompute.isEnergyOutdated({
+            amount: '105',
+            lastUpdateEpoch: 9,
+            totalLockedTokens: '5',
+        }, {
+            week: 10,
+            energy: {
+                amount: '100',
+                lastUpdateEpoch: 10,
+                totalLockedTokens: '5',
+            }
+        })).toBe(false);
+    });
+    it('current user energy values are older', () => {
+        const UserEnergyCompute = module.get<UserEnergyComputeService>(UserEnergyComputeService);
+
+        expect(UserEnergyCompute).toBeDefined();
```

Comment:
you don't need this check. Already have a unit test for it

Reply:
removed